app.R

```diff
@@ -528,16 +525,32 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
       show('gsheet_btn')
       
     } else {
-      output$text2 <- renderUI({
-        HTML("Your metadata is valid!")
-      })
-
+      
+      validation_res <- "Valid"
       ### show submit button
       output$submit <- renderUI({
         actionButton("submitButton", "Submit to Synapse")
       })
 
     }
+    }
+    
+    ### format output text
+    output$text2 <- renderUI({
+      # test if template is selected and filled manifest is uploaded
+      shiny::validate(need(!is.null(input$template_type), "Please choose a template in Select your Dataset tab !!!"),
```

Comment:
Change font color of message to red.

Error message: "Please select a template from the 'select your dataset tab'".

Reply:
update message in b1d9197
change color to red in 4d26a50